src/main/resources/public/js/app.js

```diff
@@ -82,15 +84,34 @@ for (var i = 0; i < gameModel.computerMisses.length; i++) {
 for (var i = 0; i < gameModel.computerHits.length; i++) {
    $( '#TheirBoard #' + gameModel.computerHits[i].Down + '_' + gameModel.computerHits[i].Across ).css("background-color", "red");
 }
-
 for (var i = 0; i < gameModel.playerMisses.length; i++) {
    $( '#MyBoard #' + gameModel.playerMisses[i].Down + '_' + gameModel.playerMisses[i].Across ).css("background-color", "green");
 }
 for (var i = 0; i < gameModel.playerHits.length; i++) {
    $( '#MyBoard #' + gameModel.playerHits[i].Down + '_' + gameModel.playerHits[i].Across ).css("background-color", "red");
 }
 
+/* Waits a moment for the browser to finish updating, then checks to see
+ * if a "Game Over" alert has already been thrown. If not, it alerts the
+ * the user with "Game Over" and sets "Alerted" to true.
+ */
+setTimeout( function(){
```

Comment:
I'm going to suggest that you do away with this bit. All you need from here is...
``` javascript
if (gameModel.gameOver)
    alert("Game Over.\nRefresh to play again.");
```
Except you'll want to place it elsewhere in the code. We know that the game won't have ended until after a shot has happened, so we only need to perform this check after a response is recieved from the server after performing a shot. I'll add another line comment showing you where I think it would be good to put this.